documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM2.adoc

```diff
@@ -0,0 +1,28 @@
+= Fuse Tooling - What's New in 10.3
+:page-layout: whatsnew
+:page-component_id: fusetools
+:page-component_version: 10.3.0.AM2
+:page-product_id: jbt_core
+:page-product_version: 4.5.3.AM2
+
+== Display Fuse version corresponding to Camel version proposed
+
+When creating a new project, the Camel version can be selected among a long list. Now, the Fuse version is also displayed to help choosing version corresponding to the production version that users are using/targeting.
+
+image::./images/listOfFuseVersion.png[Fuse Version also displayed in drop-down list close to Camel version]
+
+== Update validation for similar ids between component and its definition
+
+Since Camel 2.20, it is now possible to use similar ids between the component name and its definition unless the specific property "registerEndpointIdsFromRoute" is provided.
+The validation has been updated accordingly checking for Camel version and this specific attribute.
+
+To be clear, it is in this kind o case:
+[source,xml]
+----
+<from id="timer" uri="timer:timerName"/>
+----
+
+== Improved guidance in method selection for factory methods on Global Bean
+
+When selecting factory method on a Global bean, a lot of possibilities were proposed in the user interface. The list provided to users contains now only the methods that are matching constraints of the Global definition kind used (Bean or Bean Factory).
```

Comment:
The list of factory methods for a global bean is now limited to only those methods that match the constraints of the bean's global definition type (bean or bean factory).